core/src/main/kotlin/br/com/concrete/yosef/api/property/size/HeightCommand.kt

```diff
@@ -18,6 +20,21 @@ class HeightCommand : DynamicPropertyCommand {
     }
 
     override fun apply(view: View, dynamicProperty: DynamicProperty) {
-        view.layoutParams.height = dynamicProperty.value.toInt()
+        when {
```

Comment:
The first 'when' can be replaced by a normal 'if' or extract on a method, i think it'll be more readable.

Reply:
k! AS suggest it should be a when haha